ow_lander/action/ArmMoveJoint.action

```diff
@@ -4,7 +4,9 @@ int64 joint
 float64 angle
 ---
 # result
+# FIXME: this should be final_angles
```

Comment:
These refactor is just going to be out of the scope of this refactor?

Reply:
I held off in case any of our users or even our own code use this Result/Feedback and expect them to be `final_angle` and `angle`. I think of command unification as being the epic branch where interface breaks like this are more acceptable. 

@kmdalal thoughts? I can go either way on this.

Reply:
A few comments on this:
 - the refactor shouldn't change behavior or interface, so yes this needs deferral.
 - This catch needs a Jira: it either wasn't implemented to spec in the first place, or the spec changed.   I'll create that Jira, and have it cover all applicable actions.
 - The comment is a bit confusing in this PR because it's unclear whether it's in scope, as @anthonykoutroulis mentioned.  I suggest removing it since I'm creating a Jira now.

Reply:
Comments are removed. Thanks for making the ticket.